hw/ip/kmac/dv/kmac_base_sim_cfg.hjson

```diff
@@ -75,6 +75,11 @@
       uvm_test_seq: kmac_long_msg_and_output_vseq
       run_opts: ["+test_timeout_ns=10000000000"]
     }
+    {
+      name: "{variant}_sideload"
+      uvm_test_seq: kmac_sideload_vseq
+      run_opts: ["+test_timeout_ns=4000000000"]
```

Comment:
nit: consider to set this value in this value in the base test and only override it here when it needs longer time.

Reply:
will do! thanks for the suggestion.